tabular/src/autogluon/tabular/models/tab_transformer/tab_transformer_encoder.py

```diff
@@ -0,0 +1,655 @@
+# Credits for code in this script to Milan Cvitkovic,
```

Comment:
Just noticed this code now, is it used? If it isn't used, lets remove it.

Reply:
A decent portion of it is used.